exp/api/v1alpha4/azuremanagedmachinepool_types.go

```diff
@@ -21,8 +21,21 @@ import (
 	capierrors "sigs.k8s.io/cluster-api/errors"
 )
 
+const (
+	// LabelAgentPoolMode represents mode of an agent pool. Possible values include: System, User.
+	LabelAgentPoolMode = "azurecluster.infrastructure.cluster.x-k8s.io/agentpoolmode"
```

Comment:
"azurecluster" ? maybe "azuremanagedmachinepool"? or at least azuremanagedcontrolplane?

Reply:
Will change it to azuremanagedmachinepool.